documentation/examples/jsx.coffee

```diff
@@ -0,0 +1,9 @@
+renderStarRating = ({ rating, maxStars }) ->
+  <aside title={"Rating: #{rating} of #{maxStars} stars"}>
+    {for wholeStar in [0...Math.floor(rating)]
+      <Star className="wholeStar" key={wholeStar} />}
+    {if rating % 1 isnt 0
+      <Star className="halfStar" />}
+    {for emptyStar in [Math.ceil(rating)...maxStars]
+      <Star className="emptyStar" key={emptyStar} />}
+  </aside>
```

Comment:
I like the example, but note that it doesn't fit well on relatively normal sized screen:
<img width="522" alt="screen shot 2017-06-24 at 19 30 33" src="https://user-images.githubusercontent.com/1473433/27510984-f76c28dc-5913-11e7-856d-903528cd22f5.png">